app/views/find/pages/privacy.html.erb

```diff
@@ -0,0 +1,99 @@
+<% content_for :page_title, "Privacy policy" %>
```

Comment:
These page titles should probably be in translations too. Maybe we should have another translation file just for Find 🤔

Reply:
I like the idea of having translation files for each separate domain, would help keep things distinct

Reply:
We don't use translations for the page titles to a great extent anywhere else. I have included some translation links where we have shared text in the static pages.